target/linux/ar71xx/image/tiny.mk

```diff
@@ -1,12 +1,3 @@
-define Build/mkbuffaloimg
```

Comment:
Do you really want to REMOVE this from ar71xx? Looks like a copy/paste error?!

Reply:
It isn't an error. It was move from target to the generic image build code. this was it can used by all targets.

Reply:
Yes, I moved to be able to use even ath79 target.

Reply:
Ah, now I get it! Ignore my comment.